src/utils.rs

```diff
@@ -145,16 +145,16 @@ pub async fn wait_for_tx(
                 tokio::time::sleep(check_interval).await;
             }
             Err(ProviderError::StarknetError(StarknetErrorWithMessage {
-                code: MaybeUnknownErrorCode::Known(StarknetError::TransactionHashNotFound),
-                ..
-            })) => {
+                                                 code: MaybeUnknownErrorCode::Known(StarknetError::TransactionHashNotFound),
+                                                 ..
+                                             })) => {
```

Comment:
Looks like a issues with formatting, a `cargo fmt` so the gomu CI doesn't fail would be good